### PR TITLE
refactor: rename deprecated names

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 1]
```diff
@@ -2419,4 +2419,110 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
...
     },
+    {
+        from: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
+        },
+        to: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
+        },
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:2430]: mistake ? `ResizeObserverDirective` ?

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 2]
```diff
@@ -2419,4 +2419,110 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
...
         },
     },
+    {
+        from: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
+        },
+        to: {
+            name: 'WaResizeObserver',
+            moduleSpecifier: '@ng-web-api/resize-observer',
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:2429]:
```suggestion
            moduleSpecifier: '@ng-web-apis/resize-observer',
        },
        to: {
            name: 'WaResizeObserver',
            moduleSpecifier: '@ng-web-apis/resize-observer',
```

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts [hunk 3]
```diff
@@ -2419,4 +2419,110 @@ export const IDENTIFIERS_TO_REPLACE: ReplacementIdentifierMulti[] = [
...
+        },
+    },
+    {
+        from: {
+            name: 'IntersectionObserverModule',
+            moduleSpecifier: '@ng-web-api/intersection-observer',
+        },
+        to: {
+            name: 'WaIntersectionObserver',
+            moduleSpecifier: '@ng-web-api/intersection-observer',
```
Comment (nsbarsukov) [inline projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts:2439]:
```suggestion
            moduleSpecifier: '@ng-web-apis/intersection-observer',
        },
        to: {
            name: 'WaIntersectionObserver',
            moduleSpecifier: '@ng-web-apis/intersection-observer',
```

(\+ all other similar mistakes)

Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts]: fixed

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts (hunk 2 above)
Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts]: fixed

File: projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts (hunk 1 above)
Comment (splincode) [reply projects/cdk/schematics/ng-update/v4/steps/constants/identifiers-to-replace.ts]: fixed

**Review by nsbarsukov — APPROVED** [review]

**Review by vladimirpotekhin — APPROVED** [review]

**Pull request merged**